docs/conf/atlantic/2023/news/announcing-schedule.rst

```diff
@@ -0,0 +1,80 @@
+:template: {{year}}/generic.html
+
+.. post:: July 24, 2023
+   :tags: {{year}}, {{shortcode}}-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're just over six weeks away from the first {{city}} conference, and we're working hard to make sure it'll be as welcoming and fun as previous years.
+
+Today we're announcing the full presentation schedule.
+
+Conference platform
+--------------------
+
+The conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. 
+Our Welcome Wagon guide, to be released soon, will help it feel familiar before you've even arrived.
+Hopin will show you the conference schedule in your time zone, making it easy to follow the schedule.
+
+You will receive an invite to Hopin about one week before the conference.
```

Comment:
I'd probably mention something that clarifies that the registration links will be sent to the ticketholder emails, and use the opportunity to plug ticket sales again ;)